automation/components-functests.setup.sh

```diff
@@ -32,6 +32,18 @@ export KUBECONFIG=$(cluster::kubeconfig)
 make cluster-operator-push
 make cluster-operator-install
 
+# Test kubemacpool with restricted
+if [ "$COMPONENT" == "kubemacpool" ]; then
+    cluster/kubectl.sh apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: cluster-network-addons
```

Comment:
nit: we don't need to parameterize the ns name ?
anyhow if needed can be done on follow PR